src/routes.jsx

```diff
@@ -48,16 +50,21 @@ let getBlankPage = function(){
     return <div />
 }
 
+console.log(getBasePath());
+
 export const makeRoutes = (routing) => {
     return (<Route path="/" component={Container}>
-        <Route path="/home" getComponent={lazyLoadComponent(HomePage)}/>
-        <Route path="/patient" getComponent={lazyLoadComponent(PatientViewPage)}/>
-        <Route path="/datasets" getComponent={lazyLoadComponent(DatasetPage)} />
-        <Route path="/restore" component={restoreRoute}/>
-        <Route path="/testimonials" component={TestimonialsPage}/>
-        <Route path="/blank" component={getBlankPage}/>
-        <Route path="/results" getComponent={lazyLoadComponent(ResultsViewPage)} />
-        <Route path="/study" getComponent={lazyLoadComponent(StudyViewPage)} />
+                <Route path="/home" getComponent={lazyLoadComponent(HomePage)}/>
+                <Route path="/datasets" getComponent={lazyLoadComponent(DatasetPage)} />
+                <Route path="/restore" component={restoreRoute}/>
+                <Route path="/testimonials" component={TestimonialsPage}/>
+                <Route path="/blank" component={getBlankPage}/>
+                <Route path="/results" getComponent={lazyLoadComponent(ResultsViewPage)} />
+                <Route path={getBasePath()}>
```

Comment:
The reason for this is that when we use browserHistory (instead of hashHistory or memoryHistory), the paths now include the instance roots (e.g. beta, rc, etc).  Router now needs to be aware of them.